examples/onnxruntime/inceptionV3/inceptionv3.py

```diff
@@ -0,0 +1,236 @@
+    )
+
+    parser.add_argument(
+        "--verbose",
+        action="store_true",
+        required=False,
+        default=False,
+        help='Show verbose output',
+    )
+    
+    return parser.parse_args()
+
+def get_image_list_in_dir(dir):
+    proc = subprocess.run("ls",
+                          shell=True,
+                          stdout=subprocess.PIPE,
+                          cwd=dir)
+    fileList = proc.stdout.decode().split('\n')
+    fileList
+    return fileList
+
+def softmax(x):
+    """Compute softmax values for each sets of scores in x."""
+    e_x = np.exp(x - np.max(x))
+    return e_x / e_x.sum()
+
+def run_sample(session, categories, latency, inputs, topk, batch_size):
+    
+    io_binding = session.io_binding()
+    io_binding.bind_cpu_input('input', inputs.cpu().detach().numpy())
+    io_binding.bind_output('output')
+    start = time.time()
+    session.run_with_iobinding(io_binding)
+    latency.append(time.time() - start)
+    ort_outputs = io_binding.copy_outputs_to_cpu()[0]
+
+    #Get prediction for each item
+    for i in range(batch_size):
+        output = ort_outputs[i].copy().flatten()
+        output = softmax(output)  # this is optional
+        top5_catid = np.argsort(-output)[:topk]
+
+        for catid in top5_catid:
+            print(categories[catid], output[catid])
+
+def main():
+    flags = parse_input_args()
+
+    if flags.verbose:
+        print(flags)
+
+    if flags.verbose:
+        print("Reading in Imagenet classes")
+
+    # Read the categories
+    with open("./imagenet_classes.txt", "r") as f:
+        categories = [s.strip() for s in f.readlines()]
+
+    if flags.verbose:
+        print("Getting Exported Model from Torch")
+
+    # Export the model to ONNX
+    image_height = 299
+    image_width = 299
+    x = torch.randn(flags.batch, 3, image_height, image_width, requires_grad=True)
+    #torch_out = inception_v3(x)
+    torch.onnx.export(
+        inception_v3,  # model being run
+        x,  # model input (or a tuple for multiple inputs)
+        "inception_v3.onnx",  # where to save the model (can be a file or file-like object)
+        export_params=
+        True,  # store the trained parameter weights inside the model file
+        opset_version=14,  # the ONNX version to export the model to
+        do_constant_folding=
+        True,  # whether to execute constant folding for optimization
+        input_names=['input'],  # the model's input names
+        output_names=['output'])  # the model's output names
+
+    # Quantize the model
+    if flags.fp16:
+        if flags.verbose:
+            print("FP16 Quantization Enabled")
+        os.environ["ORT_MIGRAPHX_FP16_ENABLE"] = "1"  # Enable FP16 precision
+    else:
+        os.environ["ORT_MIGRAPHX_FP16_ENABLE"] = "0"  # Disable FP32 precision
+
+    session_ops = onnxruntime.SessionOptions()
+    if flags.verbose:
+        session_ops.log_verbosity_level = 0
+        session_ops.log_severity_level = 0
```

Comment:
level = 0 does it print anything ? isn't it default ?

Reply:
Default is 2. Onnxruntime specifies verbosity levels and severity here: https://onnxruntime.ai/docs/api/python/api_summary.html#onnxruntime.SessionOptions.log_severity_level

its meant to print and see where the nodes go and how the model is invoked along with any other events through Onnxruntime. We can still enable MIGRAPHX_TRACE_EVAL to verify that the script does infact run as fp16 or fp32 eg

MIGRAPHX_TRACE_EVAL=1 python3 inceptionv3.py

Reply:
I meant for `verbosity_level = 0` not severity

Reply:
Its so we ensure the verbosity_level=0. I don't want to assume otherwise/defaults.